### ISSUE TITLE
Steve's carts doesnt handle new randomgen bee hives

### ISSUE THREAD

**Issue by Saereth** [CreeperHost/StevesCarts2#22]
Using the cart for a tree farm will eventually end up with a field of bee hives as it doesnt have any way currently to deal with them. Imagine included for clarity:
![2024-05-10_13 54 42](https://github.com/CreeperHost/StevesCarts2/assets/9200909/604c9f69-4d22-4ff4-b391-0826b6caa2a7)

**Comment by Saereth** [comment]
actually now that I think about this you could just remove all the flowers to prevent this so probably a non issue

**Comment by gigabit101** [comment]
Yeah this is not something the wood-cutting module should handle however a new module that can be added with the tree cutter module would make sense to harvest these

**Comment by gigabit101** [comment]
Going to merge this with #21

**Comment by gigabit101** [comment]
I think it's about time we just redid the tree-cutting logic, It is ancient ported code

**Comment by brandon3055** [comment]
For now, the cart will just break any hives attached to the trees it harvests.

**Comment by gigabit101** [comment]
New versions now available on CurseForge and Modrinth
